.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "typescript.tsdk": "node_modules/typescript/lib"
```

Comment:
VSCode uses latest TS version in the editor. its good to have the same package.json version in VSCode